RocketChat/Rocket.Chat.ReactNative
# Chore: Clean InAppNotification - TypeScript

## Proposed changes
Clean InAppNotification - TypeScript

## Issue(s)	


## How to test or reproduce
With the App open on the list of chats, send a message from another device and check if the in app notification will shows up normally.

## Screenshots

## Types of changes




## Checklist



## Further comments
